src/app/components/available-balance.tsx

```diff
@@ -20,7 +21,7 @@ export function AvailableBalance(props: { balance: Money; balanceTooltipLabel?:
         </styled.span>
         <BasicTooltip label={balanceTooltipLabel} side="top">
           <Box>
-            <InfoIcon color="accent.text-subdued" size="xs" />
+            <InfoCircleIcon style={{ color: token('colors.accent.text-subdued') }} width="xs" />
```

Comment:
Curious if we need to use `token` here as sometimes we do and sometimes not?

Reply:
I guess this is something to do with dynamic styling? 🐼

Reply:
Yeah, struggling with this a bit sometimes, but I think it has to be used bc the underlying component receiving it is a none panda pattern element. It is a React `path`? All I know is, it wasn't working until I added it, but I'll double back and check today.

Reply:
I was able to do some refactoring and got this to work w/out the token. 🤞

Reply:
That's great! I don't mind using it sometimes but I rarely understand why, even if I am doing it myself. It makes sense RE dynamic styling but maybe we can do it better